conf/openmetadata.yaml

```diff
@@ -165,6 +165,24 @@ elasticsearch:
   socketTimeoutSecs: ${ELASTICSEARCH_SOCKET_TIMEOUT_SECS:-60}
   batchSize: ${ELASTICSEARCH_BATCH_SIZE:-10}
 
+kafka:
+  topics: ${KAFKA_TOPICS:-[]}
```

Comment:
there should only be one topic

Reply:
we should take the minimum required configs, such as bootstrap servers and topic. Rest of them should be key/value pairs that users can chose what they want to pass